ensure_dependencies.py

```diff
@@ -304,6 +304,9 @@ def resolve_deps(repodir, level=0, self_update=True, overrideroots=None, skipdep
     parenttype = get_repo_type(repodir)
     _root = config.get("_root", {})
 
+    if parenttype is None:
```

Comment:
Instead adding two new lines, you could simply return `'hg'` instead `None` from `get_repo_types()`.